app/apps/server/communication/uikit.js

```diff
@@ -39,17 +41,20 @@ router.use((req, res, next) => {
 		'x-auth-token': authToken,
 	} = req.headers;
 
-	if (!userId || !authToken) {
-		return unauthorized(res);
+	const { token: visitorToken } = req.body;
```

Comment:
is there a particular reason to use the name `token` instead of `visitorToken`? I think `token` is too broad.. also any reason this is not on request headers as well?

Reply:
I followed the same approach from other Livechat APIs - `token` field in the body of the request. That was the only reason

Reply:
for ui-kit endpoint I'd like to keep auth info in request headers.. so I think it should be moved to a header like `x-visitor-token`

Reply:
right, this will depend on a change to the https://github.com/RocketChat/Rocket.Chat.Livechat repo then